subsys/net/l2/openthread/openthread.c

```diff
@@ -397,9 +401,11 @@ int openthread_stop(struct openthread_context *ot_context)
 
 	openthread_api_mutex_lock(ot_context);
 
-	error = otThreadSetEnabled(ot_context->instance, false);
-	if (error == OT_ERROR_INVALID_STATE) {
-		NET_DBG("Openthread interface was not up [%d]", error);
+	if (!IS_ENABLED(CONFIG_OPENTHREAD_RCP)) {
```

Comment:
Same here, I'd rather exit early with the same condition as in `openthread_start()`.

Reply:
Same as above.